udata/core/spatial/commands.py

```diff
@@ -65,20 +66,24 @@ def load(filename, drop=False):
     with open(zones_filepath) as fp:
         unpacker = msgpack.Unpacker(fp, encoding=str('utf-8'))
         for i, geozone in enumerate(unpacker, start=1):
```

Comment:
Is the `enumerate` and the `i` variable useful ? I think it can be safely removed